Add GeoIP Web Service docs

wesrice opened the pull request.

kevcenteno commented on content/geoip/docs/web-services/_examples/geolite.json:
```diff
@@ -0,0 +1,9 @@
+curl -X POST \
```
This file isn't needed

kevcenteno commented on content/geoip/docs/web-services.mdx:
```diff
@@ -0,0 +1,172 @@
+
+<Alert type="info">
+  If you are a legacy GeoIP customer, please see our
+  [What's New in GeoIP2](https://dev.maxmind.com/geoip/geoip2/whats-new-in-geoip2/)
```
Can we add a TODO message here?

kevcenteno replied:
Or maybe remove this alert

kevcenteno commented on content/geoip/docs/web-services/requests.mdx:
```diff
@@ -0,0 +1,108 @@
+| City     | `POST`      | `https://geoip.maxmind.com/geoip/v2.1/city/{ip_address}`     |
+| Insights | `POST`      | `https://geoip.maxmind.com/geoip/v2.1/insights/{ip_address}` |
+
+The **geoip.maxmind.com** hostname automatically picks the data center
```
Let's wrap this in backticks instead of `**`

kevcenteno commented on content/geoip/docs/web-services/requests.mdx:
```diff
@@ -0,0 +1,108 @@
+| Country  | `POST`      | `https://geolite.info/geoip/v2.1/country/{ip_address}`  |
+| City     | `POST`      | `https://geolite.info/geoip/v2.1/city/{ip_address}`     |
+
+The **geolite.info** hostname automatically picks the data center
```
Let's wrap this in backticks instead of `**`

kevcenteno reviewed (changes requested):
Just a few small changes.  Also, there's a build failure

wesrice commented:
@kevcenteno I cleaned this up.

There was an issue with our build script (CSP css file generation) that I fixed. I noticed a new warning output from Gatsby.

```
warn Unsafe builtin method was used, future builds will need to rebuild all pages
warn WebpackError: Unsafe builtin usage fs.writeFileSync
    at dev-site/src/gatsby/gatsby-ssr/replaceRenderer.tsx:61:3
    at dev-site/.cache/api-runner-ssr.js:57:41
    at dev-site/.cache/api-runner-ssr.js:52:25
    at dev-site/.cache/static-entry.js:247:5
    at dev-site/node_modules/@babel/runtime/helpers/interopRequireWildcard.js:19:1
```

The build still completes successfully, but this is something to keep an eye on and potentially revisit.
